src/components/exercise/group.cjsx

```diff
@@ -1,12 +1,20 @@
 React = require 'react'
 _ = require 'underscore'
-
+BS = require 'react-bootstrap'
 camelCase = require 'camelcase'
 
 ChapterSectionMixin = require '../chapter-section-mixin'
 ExerciseIdentifierLink = require '../exercise-identifier-link'
+
 DEFAULT_GROUP =
   show: false
+REVIEW_GROUP =
+  show: true
+  label: 'Review'
+  tooltip: '''Did you know? Research shows you can strengthen your memory —
+    and spend less time studying — if you revisit material over multiple study sessions.
+    OpenStax Concept Coach will include review questions from prior sections to give your
```

Comment:
I think this changes depending on if the component is used in CC or in Tutor (see mockups), no?